glotaran/builtin/io/yml/utils.py

```diff
@@ -1,4 +1,4 @@
-"""Utility functionality module for ``glotaran.builtin.io.yml.yml``"""
+"""Utility functionality module for ``glotaran.builtin.io.yml.yml``."""
```

Comment:
```suggestion
"""Utility module for ``glotaran.builtin.io.yml.yml``."""
```
'Functionality' doesn't add anything here, just 'utility module' or "Yaml utility module".